src/Tizen.TV.UIControls.Forms/Handler/AccessKeyEffect.cs

```diff
@@ -59,10 +58,10 @@ Page GetParentPage()
             return null;
         }
 
-        void OnKeyDown(object sender, EcoreKeyEventArgs e)
+        void OnKeyDown(object sender, Tizen.NUI.Window.KeyEventArgs e)
```

Comment:
I'd like to suggest to use alias such as `TWindow` in `RemoteKeyEventEffect` class, it can be improve code consistency.

Reply:
Sounds great! Thanks for the opinion!

Reply:
Updated!